openquake/server/views.py

```diff
@@ -906,7 +914,9 @@ def aristotle_run(request):
     for key in ['dip', 'strike']:
         if key in rupdic and rupdic[key] is None:
             del rupdic[key]
-    if not os.path.isfile(station_data_file):
+    if 'error' in rupdic:
+        del rupdic['error']
```

Comment:
Can you add a comment about why we are removing the error?

Reply:
You are right, it is not needed. I am fixing that.